src/clients/lan.rb

```diff
@@ -160,6 +160,11 @@ def main
             "help" => _("Bond Ports"),
             "type" => "string"
           },
+          "slaves"   => {
+            # Commandline option help
+            "help" => _("Bond Ports (obsolete, use bond_ports instead)"),
```

Comment:
This will confuse the hell out of translators, because they don't see the code.
Add a comment please